src/test/java/org/fcrepo/integration/auth/webac/WebACRecipesIT.java

```diff
@@ -179,6 +179,17 @@ public void scenario1() throws IOException {
             assertTrue("Missing Link header", header.isPresent());
         }
 
+        logger.debug("Can username 'smith123' read a child of {}", testObj);
```

Comment:
Just me being picky, but please log assertions for tests, e.g. "Thus-and-such kind of request should include information about whether the user can read thus-and-such a resource." or whatever is appropriate. The half-question is confusing if you don't understand what is under test.